versions.lock

```diff
@@ -5,8 +5,10 @@ aopalliance:aopalliance:1.0 (1 constraints: 170a83ac)
 asm:asm:3.1 (2 constraints: 4f19c3c6)
 com.carrotsearch:hppc:0.7.2 (1 constraints: f70cda14)
 com.clearspring.analytics:stream:2.7.0 (1 constraints: 1a0dd136)
+com.esotericsoftware:kryo:4.0.2 (1 constraints: 720da324)
```

Comment:
Is there a way to substitute `kryo` with `kryo-shaded`?
I took a quick stab at gradle dependency substitution but couldn't get it going because `--write-locks` still output kryo, not kryo-shaded.